.github/workflows/integration-tests.yml

```
@@ -84,6 +84,9 @@ jobs:
       fail-fast: false
       matrix:
         product:
+          - name: automation
+            nodes: 27
+            os: ubuntu20.04-32cores-128GB
```

Since you are only adding 6 tests this probably doesn't need the ability to run 27 tests in parallel and probably doesn't need a specified large node.
I recommend lowering it so we aren't throwing too many resources at it.
```suggestion
          - name: automation
            nodes: 6
            os: ubuntu-latest
```